lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -226,6 +220,14 @@ abstract class AssetPickerBuilderDelegate<Asset, Path> {
   AssetPickerTextDelegate get semanticsTextDelegate =>
       Singleton.textDelegate.semanticsTextDelegate;
 
+  Iterable<SpecialItem> get prepandSpecialItems => specialItems.where(
+        (model) => model.itemPosition == SpecialItemPosition.prepend,
+      );
+
+  Iterable<SpecialItem> get appendSpecialItems => specialItems.where(
+        (model) => model.itemPosition == SpecialItemPosition.append,
+      );
```

Comment:
Could they be final to avoid dynamic reference changes?

Reply:
done update to final.